content/topics/Analyze/causal-inference/introduction/causal-inference-intro.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "The Concept and Motivation of Causal Inference"
+description: "This topic motivates causal inference and provides theoretical background for all the practical methods you can find in this section."
+keywords: "causal, inference, econometrics, regression, model, potential, outcomes, framework, treatment, effect, control, ATE, ATT"
+draft: false
+weight: 1
+author: "Valerie Vossen"
+aliases:
+  - /causal-inference
+  - /causal-inference-introduction
+  - /causal-inference-intro
+---
+
+## Overview
+
+Causal inference nowadays plays a very important role in quantitative social sciences. It is about estimating the impact of variables on specific outcomes, by understanding the causal relationship. 
+Unlike correlation, which only indicates a statistical relationship between variables, causal inference aims to determine if one variable directly influences another. It's essential to recognize that correlation does not imply causation; just because two variables are correlated does not mean that one causes the other. Additionally, causal relationships may exist without observable correlations. 
+
+<p align = "center">
+<img src = "../images/corr_cause.png" width="700">
+<figcaption> Source: xkcd (https://xkcd.com/552/) </figcaption>
```

Comment:
image is not showing up when I preview. Double check?

Reply:
Yes, this is a common error when previewing, should work on the website when merged. But I will double-check when page is online.